commands/scanpullrequest.go

```diff
@@ -56,7 +61,15 @@ func scanPullRequest(params *utils.FrogbotParams, client vcsclient.VcsClient) er
 	// Comment frogbot message on the PR
 	getTitleFunc, getSeverityTagFunc := getCommentFunctions(params.SimplifiedOutput)
 	message := createPullRequestMessage(vulnerabilitiesRows, getTitleFunc, getSeverityTagFunc)
-	return client.AddPullRequestComment(context.Background(), params.RepoOwner, params.Repo, message, params.PullRequestID)
+	err = client.AddPullRequestComment(context.Background(), params.RepoOwner, params.Repo, message, params.PullRequestID)
+	if err != nil {
+		return err
+	}
+	// Fail the Frogbot task if a security issue was found
```

Comment:
// Fail the Frogbot task if a security issue was found
-->
// Fail the Frogbot task, if a security issue is found and Frogbot isn't configured to avoid the failure.